circuit_maintenance_parser/parser.py

```diff
@@ -175,10 +175,10 @@ def process(self) -> Iterable[Maintenance]:
             return result
 
         except ValidationError as exc:
-            raise MissingMandatoryFields from exc
+            raise MissingMandatoryFields(exc)  # pylint: disable=W0707
 
         except Exception as exc:
-            raise ParsingError from exc
+            raise ParsingError(exc)  # pylint: disable=W0707
```

Comment:
Can you explain the reason for this change? Pylint certainly seems to think it's a step in the wrong direction. If it's just to make it easier for logging purposes, note that `raise X from Y` will attach `Y` under `X.__cause__` so you can access it that way if desired.

Reply:
Will double-check it